src/main/java/com/sprints/repository/SprintsValidationsRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.sprints.repository;
+
+import com.sprints.model.Sprint;
+
+public interface SprintsValidationsRepository {
+	public Sprint sprintRepoValidationActive();
```

Comment:
maybe we could rename the method for a more specific action like: "getFirstActiveSprint" something like that.

Reply:
how about oneSprintActiveValidation?

Reply:
yes, that name will work !